changelog/fragments/helm-upgrade-rename.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      In Helm-based operators, usages of the verb `Update` in the logs were renamed to `Upgrade` to better align with Helm nomenclature.
+    kind: "change"
+    breaking: false
```

Comment:
After we spoke about it, could you please remove the fragment since this change does not really impact the users? (just the logs)